identity-service/scripts/run-tests.sh

```diff
@@ -58,7 +65,13 @@ elif [ -x "$(command -v psql)" ]; then
 fi
 
 # tests
-./node_modules/mocha/bin/mocha test/index.js --timeout 10000 --exit
+if [ "${debug}" ]; then
+  # If the -d debug flag is provided, run the tests with no timeout so that
+  # debugging does not get interrupted
+  ./node_modules/mocha/bin/mocha test/index.js --timeout 0 --exit
```

Comment:
could potentially DRY this up with setting bash var `TIMEOUT` to 0 if debug and 10000 if not